Dragonflight/RogueAssassination.lua

```diff
@@ -272,16 +272,11 @@ local calculate_multiplier = setfenv( function( spellID )
 
     if stealth then
         if talent.nightstalker.enabled then
-            mult = mult * 1.5
-        end
-
-        -- Garrote.
-        if talent.subterfuge.enabled and spellID == 703 then
-            mult = mult * 1.8
+            mult = mult * 1.08
         end
     end
 
-    if FindPlayerAuraByID( 392401 ) and spellID == 703 then
+    if FindPlayerAuraByID( 392401 ) or FindPlayerAuraByID( 392403 ) and spellID == 703 then
```

Comment:
Need parentheses around the FindPlayerAuraByID calls:

`if ( FindPlayerAuraByID( 392401 ) or FindPlayerAuraByID( 392403 ) ) and spellID == 703 then`